axelor-contract/src/main/java/com/axelor/apps/contract/service/ContractService.java

```diff
@@ -42,7 +48,7 @@
    * @param date
    */
   @Transactional(rollbackOn = {AxelorException.class, Exception.class})
```

Comment:
No need to specify rollbackOn here
See https://github.com/google/guice/wiki/Transactions